### PR TITLE
Feature/msbuild create

### PR DESCRIPTION
**Pull request by memsharded**
Close https://github.com/conan-io/docs/issues/3065

### CODE REVIEW

File: examples/tools/microsoft/msbuild/create_your_first_package.rst [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                      compiler.runtime_type: Release
+                      compiler.version: 193         
+                      os: Windows                   
+         
+               
+
+
+
+
+.. seealso::
```
Comment (memsharded) [inline examples/tools/microsoft/msbuild/create_your_first_package.rst:165]: I think we should change all ReadMore for ``seealso``:

- The ReadMore section sometimes slips in Table of Contents, and it results very ugly
- It doesn't allow to introduce more inline links, only at the very end of the section, while sometime some more inline links at the end of a subsection would be useful

File: tutorial/creating_packages/create_your_first_package.rst [hunk 2]
```diff
@@ -88,6 +88,8 @@ Let's have a look at the package recipe *conanfile.py*:
           cmake_layout(self)
 
       def generate(self):
+          deps = CMakeDeps(self)
```
Comment (memsharded) [inline tutorial/creating_packages/create_your_first_package.rst:91]: This is now part of the template ``conan new``

File: tutorial/creating_packages/create_your_first_package.rst [hunk 3]
```diff
@@ -318,7 +338,6 @@ An **important** note: the Conan cache is private to the Conan client - modifyin
 Read more
 ---------
 
-- :ref:`Conan list command reference<reference_commands_list>`.
-- :ref:`Create your first Conan package with Autotools<examples_tools_autotools_autotools_toolchain_build_project_autotools_toolchain>`.
-- :ref:`Create your first Conan package with Meson<examples_tools_meson_toolchain_build_simple_meson_project>`.
-- :ref:`Create your first Conan package with CMake<examples-tools-cmake-toolchain-build-project-presets>`.
```
Comment (memsharded) [inline tutorial/creating_packages/create_your_first_package.rst:324]: These were confusing, they are pointing to "consumer" projects, not to package creation.

File: examples/tools/microsoft/msbuild/create_your_first_package.rst [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+        copy(self, "*.h", os.path.join(self.source_folder, "include"),
+             dst=os.path.join(self.package_folder, "include"))
+        copy(self, "*.lib", src=self.build_folder, dst=os.path.join(self.package_folder, "lib"),
+             keep_path=False)
+
+
+Let's explain the different sections of the recipe briefly:
+
+- Note there are no ``options`` like the ``shared`` option in this recipe. The current project
+  always build a static library, so it is not optional.
```
Comment (RubenRBS) [inline examples/tools/microsoft/msbuild/create_your_first_package.rst:78]:
```suggestion
  always builds a static library, so it is not optional.
```

File: examples/tools/microsoft/msbuild/create_your_first_package.rst [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+- The ``layout()`` defines a typical VS layout, this is less flexible than a CMake one, so it
+  doesn't allow any parametrization.
+- The ``generate()`` method calls ``MSBuildToolchain`` to generate a ``conantoolchain.props`` 
+  file, that the project must add to his properties. If the project had dependencies with Conan
```
Comment (RubenRBS) [inline examples/tools/microsoft/msbuild/create_your_first_package.rst:82]:
```suggestion
  file, that the project must add to its properties. If the project had dependencies with Conan
```

**Review by RubenRBS — APPROVED** [review]

**Review by czoido — APPROVED** [review]

**Pull request merged**